### PR TITLE
Add review comment notify

### PR DESCRIPTION
**Pull request by echoulen**

### CODE REVIEW

File: .github/workflows/action-test.yml [hunk 1]
```diff
@@ -1,10 +1,9 @@
 name: "notify-test"
 on:
   pull_request:
-    types:
-      - opened
-      - reopened
-      - closed
+    types: [opened, reopened, closed]
+  pull_request_review_comment:
+    types: [created]
```
Comment (echoulen) [inline .github/workflows/action-test.yml]: test

**Review by echoulen — COMMENTED** [review]
asgsagasg

**Review by echoulen — COMMENTED** [review]
safassag

File: .github/workflows/action-test.yml [hunk 2]
```diff
@@ -1,10 +1,9 @@
 name: "notify-test"
 on:
   pull_request:
-    types:
-      - opened
-      - reopened
-      - closed
+    types: [opened, reopened, closed]
```
Comment (Nextdrive-CarlosLi) [inline .github/workflows/action-test.yml:4]: hello

**Review by Nextdrive-CarlosLi — CHANGES_REQUESTED** [review]

**Pull request closed without merging**